### PR TITLE
moved sources into src folder

### PR DESCRIPTION
**Pull request by jjr840430**

### CODE REVIEW

**Comment by jjr840430** [comment]
@drystone Can you help me to check issues from travis checking?

File: src/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,77 @@
+
+ACLOCAL_AMFLAGS = -I m4
```
Comment (drystone) [inline src/Makefile.am]: Could it be ../m4 ?

**Comment by drystone** [comment]
`make check` is failing after `./configure --with-cmocka`

**Comment by jjr840430** [comment]
@drystone Fixed, There is a wrong link for libretrace.so in cmockatest.c :)

**Comment by jjr840430** [comment]
@ronaldtse Implemented #263 . Please review and merge.

**Comment by jjr840430** [comment]
related with issue #260

**Comment by ronaldtse** [comment]
Thanks @jjr840430 !

**Pull request merged**